Sprint-1/errors/4.js

```diff
@@ -1,2 +1,5 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const Hour12ClockTime = "20:53";
+const Hour24ClockTime = "08:53";
```

Comment:
This is something extra. Did you notice the variable names do not quite match the value assigned to the variables?